apps/docs/pages/guides/platform/shared-responsibility-model.mdx

```diff
@@ -34,6 +34,10 @@ You can get away with a poorly-designed database for a while by simply adding co
 
 If you don’t want to implement logic inside your database, that is 100% fine. You can use _any_ tools which work with Postgres.
 
+## You are responsible for third-party services
+
+Supabase offers a lot of opportunities for flexibly integrating with third-party services. This varies from making calls to external APIs from Postgres triggers, to using custom SMTP services for greater control over your outgoing emails. You are free to use and integrate with any service, but you're also responsible for ensuring that the performance and availability of the services you use match up with your application's requirements.
```

Comment:
```suggestion
Supabase offers a lot of opportunities for flexibly integrating with third-party services, such as:

- OAuth and SAML login providers
- SMTP and SMS sending APIs 
- Calls to external APIs within Postgres functions or triggers
- Various integrations with third-party services

You are free to use and integrate with any service, but you're also responsible for ensuring that the performance and availability of the services you use match up with your application's requirements. We don't actively monitor for outages or security vulnerabilities with many third-party services, so you should actively take measures to monitor logs and communication channels.
```

Wanted to include the bit about security vulnerabilities, and enumerate more specifically what APIs we're referring to.

Reply:
> integrating with third-party services

> - Various integrations with third-party services
	
This sounds somewhat off, but other than that I definitely agree we should add it here.